python/ray/dashboard/client/src/pages/dashboard/tune/TuneTensorBoard.tsx

```diff
@@ -0,0 +1,64 @@
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import createStyles from "@material-ui/core/styles/createStyles";
+import withStyles, { WithStyles } from "@material-ui/core/styles/withStyles";
+import { connect } from "react-redux";
+import { StoreState } from "../../../store";
+import { dashboardActions } from "../state";
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      backgroundColor: theme.palette.background.paper,
+      padding: theme.spacing(2),
+      "& > :not(:first-child)": {
+        marginTop: theme.spacing(4)
+      }
+    },
+    board: {
+      width: "100%",
+      height: "600px",
+      border: "none"
+    },
+    warning: {
+      fontSize: "0.8125rem"
+    }
+  });
+
+const mapStateToProps = (state: StoreState) => ({
+  error: state.dashboard.error
+});
+
+const mapDispatchToProps = dashboardActions;
+
+class TuneTensorBoard extends React.Component<
+  WithStyles<typeof styles> &
+    ReturnType<typeof mapStateToProps> &
+    typeof mapDispatchToProps
+> {
+  render() {
+    const { classes, error } = this.props;
+
+    return (
+      <div className={classes.root}>
+        {error === "TypeError: Failed to fetch" && (
+          <Typography className={classes.warning} color="textSecondary">
+            Warning: Tensorboard server closed. View Tensorboard by running
+            "tensorboard --logdir" if not displaying below.
+          </Typography>
+        )}
+        <iframe
+          src="http://localhost:6006/"
```

Comment:
Will it always be this port?

Reply:
I think you may also need to add a disclaimer that this only work on local machine/cluster

Reply:
It will be port 6006 by default. It is possible to use a custom port if we add another command line argument when we start the server in dashboard.py